source/install/desktop-app-install.rst

```diff
@@ -202,6 +200,16 @@ The following additional documentation resources are also available for the Matt
 Troubleshooting your Desktop App installation
 ----------------------------------------------
 
+Where is configuation stored locally?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The location of the Mattermost desktop app configuration file depends on the platform where you're running Mattermost:
+
+- Windows: ``Users\<username>\AppData\Roaming\Mattermost``
+- macOS DMG: ``/Users/<username>/Library/Application Support/Mattermost``
+- macOS TestFlight: ``/Users/<username>/Library/Containers/Mattermost/Data/Library/Application Support/Mattermost``
```

Comment:
> @devinbinnie - Is there a separate path for macOS via the App Store? Or would that be the same path as TestFlight?

App Store and TestFlight should be the same.

> Also - Do users refer to the "DMG version" using other terminology? Will "DMG" be understood widely enough?

Not sure exactly, I call it that because of the file name. Another way could be to call it the 'standalone installer'. That might be better. cc @nab-77 if you have an opinion.